src/Core/Block.cpp

```diff
@@ -403,7 +404,12 @@ Block Block::sortColumns() const
 {
     Block sorted_block;
 
+    /// std::unordered_map cannot be used to guarantee the sort order
```

Comment:
Absolutely right.
It looks like someone was replaced std::map to std::unordered_map without paying attention.

Reply:
>without paying attention.

Let's blame lack of test :)